internal/olm/operator/bundle/install.go

```diff
@@ -46,6 +47,7 @@ func NewInstall(cfg *operator.Configuration) Install {
 }
 
 func (i *Install) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&i.LocalBundle, "local-bundle", "", "bundle directory or image to extract package information from. If unset, the bundle image arg is used")
```

Comment:
Can this optional flag strictly be bundle directory locally that is used for extracting package info from? I don't see the value of passing an image to this value, considering that the command takes bundle image as a required positional argument.